packages/esm-appointments-app/src/appointments/common-components/appointments-actions.component.tsx

```diff
@@ -70,27 +65,10 @@ const AppointmentActions: React.FC<AppointmentActionsProps> = ({ visits, appoint
           </Button>
         );
       case isTodayAppointment: {
-        const isAfterNoon = new Date().getHours() > 12;
-
-        if (scheduleType === 'Pending' && isAfterNoon) {
-          return (
-            <Button onClick={handleOpenDefaulterForm} size="sm" kind="tertiary">
-              {followUpButtonText}
-            </Button>
-          );
-        }
-
```

Comment:
I'm not in a position to say whether this is right or not, but this does seem to indicate a point where we have implementation needs for extensibility that's not currently provided.